src/Tribe/Attendee_Repository.php

```diff
@@ -672,7 +673,7 @@ public function filter_by_checkedin( $checkedin ) {
 			return Tribe__Repository__Query_Filters::meta_in( $meta_keys, '1', 'is-checked-in' );
 		}
 
-		return Tribe__Repository__Query_Filters::meta_not_in_or_not_exists( $meta_keys, '1', 'is-not-checked-in' );
+		$this->filter_query->meta_not( $meta_keys, '1', 'is-not-checked-in' );
```

Comment:
Should we be returning this?
```suggestion
		return $this->filter_query->meta_not( $meta_keys, '1', 'is-not-checked-in' );
```

Reply:
Nope, this is handled within the `meta_not` method which directly modifies the query and doesn't need to be returned.